lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -406,6 +406,30 @@ protected function transformReportbackItem($data) {
   }
 
   /**
+   * Transform Signup data and prepare for API response.
+   *
+   * @param  object  $data Signup object.
+   *   An object containing properties of Signup data:
+   *   - id: (string) The Signup id.
+   *   - created_at: (string) Date Signup was created.
+   *   - campaign: (string) Campaign Signup belongs to.
+   *   - campaign_run: (string) Campaign run Signup belongs to.
+   * @return array
+   */
+  protected function transformSignup($data) {
+    return [
+      'signup_id' => $data->id,
+      'source' => $data->source,
+      'created_at' => $data->created_at,
+      'campaign' => [
```

Comment:
To clarify the above - should we include which particular `campaign_run` the `signup` is associated with? The campaign object returns the `current campaign_runs` but not sure if this is necessarily the one associated with the `signup`.